fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -105,6 +105,12 @@ def self.mac?
       (/darwin/ =~ RUBY_PLATFORM) != nil
     end
 
+    # @return CPU type
+    def self.cpu_type
+      return "" unless self.mac?
+      return `sysctl -n machdep.cpu.brand_string`.split("\n").first
```

Comment:
Instead of CPU type, do we actually (or also) want the architecture type? This might be good to see if its running in Rosetta? This can be checked with `uname -p` or `arch`. Thoughts? 😇

Reply:
Hi @joshdholtz !
You are completely right. Information about the type of cpu is completely unnecessary if we know what the architecture is. On the other hand, maybe some users may find this information (CPU Info) sensitive, and since we don't need it, I replace it architecture type.

`uname -p` returns a lot of information that is not needed (at least at this point 😄 ), so I changed the command to `arch` which returns only type eg. `i386`

Thanks! 🚀

Reply:
Actually `uname -p` returns the same thing as `arch`. I used some other commando before 🤣. Anyway it works and PR is updated.